linux_mac/magpi-issue-downloader.sh

```diff
@@ -10,49 +10,37 @@
 # VERSION=0.1.2
 # USAGE="Usage: sh magpi-issue-downloader.sh [-f firstissue] [-l lastissue]"
 
-if [ ! -d "issues" ]; then
- mkdir issues
+MAGPI_URL="https://magpi.raspberrypi.com"
+BASEDIR=`dirname $0`/..
+OUTDIR=$BASEDIR/issues
+
+if [ ! -d "$OUTDIR" ]; then
+ mkdir $OUTDIR
 fi
 
 i=1
-file="../issues.txt";
+file="$BASEDIR/issues.txt";
 issues=$(cat "$file");
 
-if [ -z $1 ]
-then
-	wget https://www.raspberrypi.org/magpi-issues/ \
-	--recursive \
-	--no-directories \
-	--no-parent \
-	--directory-prefix issues \
-	--mirror \
-	--accept-regex "/MagPi[0-9]+\.pdf" \
-	--reject index.html \
-	--execute robots=off
-
-else
-
 	while :
 	do
 		case "$1" in
 		-f) shift; i="$1";;
 		-l) shift; issues="$1";;
 		--) shift; break;;
-		-*) usage "bad argument $1";;
+		-*) echo "usage: bad argument $1";;
 		*) break;;
 		esac
 		shift
 	done
 
 	while [ $i -le $issues ]
 	do
-		if [ "$i" -lt 10 ]; then
-			wget https://www.raspberrypi.org/magpi-issues/MagPi0$i.pdf -P issues/
-
-		else
-			wget https://www.raspberrypi.org/magpi-issues/MagPi$i.pdf -P issues/
-		fi
+		printf -v page_url "$MAGPI_URL/issues/%d/pdf" $i
```

Comment:
I already moved everything from here into the downloader itself
https://github.com/joergi/downloader/blob/main/linux_mac/downloader.sh